src/neuroconv/datainterfaces/text/csv/csvtimeintertervalsinterface.py

```diff
@@ -0,0 +1,9 @@
+import pandas as pd
+
+from ..timeintervalsinterface import TimeIntervalsInterface
+from ....utils.types import FilePathType, Optional
+
+
+class CsvTimeIntervalsInterface(TimeIntervalsInterface):
+    def _read_file(self, file_path: FilePathType, **read_kwargs):
+        return pd.read_csv(file_path, **read_kwargs)
```

Comment:
This structure starts to lend itself to a much more general base I/O class, similar to how the `GenericDataChunkIterator` requires specification of `_get_data` to tell it 'how to read the data' from something initialized during `__init__`

I can also foresee a case of building something like a `MatlabTimeIntervalsInterface` that defines the `_read_file` in terms of scipy or hdfstorage or just vanilla h5py commands

Not saying we need to do that in this PR, just thinking ahead

Reply:
Yes, the idea is this structure would allow for reading many different kinds of formats. Hierarchical formats like HDF5 and .mat files might be tricky, but surely we could support some common patterns in the future. For now, I think supporting csv, xls, and xlsx is a good scope for this PR.